src/sentry/api/endpoints/project_platforms.py

```diff
@@ -11,8 +11,22 @@ class ProjectPlatformSerializer(Serializer):
     def serialize(self, obj, attrs, user):
         return {'platform': obj.platform, 'dateCreated': obj.date_added}
 
+ERR_FIELD_REQUIRED = 'This field is required.'
+
 
 class ProjectPlatformsEndpoint(ProjectEndpoint):
     def get(self, request, project):
         queryset = ProjectPlatform.objects.filter(project_id=project.id)
         return Response(serialize(list(queryset), request.user))
+
+    def put(self, request, project):
```

Comment:
@mattrobenolt  I struggled with trying to decied whether to use put or post. The difference is not clear to me after reading a bit about both.

Reply:
You should use POST. POST is for creating new, and PUT is for updating an existing one. Generally.